packages/playground/src/components/webthree-connector/webthree-connector.tsx

```diff
@@ -11,23 +13,77 @@ export class Web3Connector {
   @Prop() accountState: AccountState = {} as AccountState;
   @Prop() networkState: NetworkState = {};
 
-  async componentDidLoad() {
+  getCurrentAddress() {
+    return (window as any).web3.eth.accounts[0];
```

Comment:
@snario You can avoid the `as any` cast using index references

```
return window["web3"].eth.accounts[0];
```

Also applies to functions below.